providers/celery/src/airflow/providers/celery/cli/celery_command.py

```diff
@@ -154,10 +154,15 @@ def worker(args):
     # This needs to be imported locally to not trigger Providers Manager initialization
     from airflow.providers.celery.executors.celery_executor import app as celery_app
 
+    # backwards-compatible: https://github.com/apache/airflow/pull/21506#pullrequestreview-879893763
+    celery_log_level = conf.get("logging", "CELERY_LOGGING_LEVEL")
+    if not celery_log_level:
+        celery_log_level = conf.get("logging", "LOGGING_LEVEL")
+
     if AIRFLOW_V_3_0_PLUS:
         from airflow.sdk.log import configure_logging
 
-        configure_logging(output=sys.stdout.buffer)
+        configure_logging(output=sys.stdout.buffer, log_level=celery_log_level)
```

Comment:
Hmm, celery_log_level used to just change the logging level in celery itself. Does this impact the Airflow side now too?

Reply:
Well what I observed is when preparing the demo, I was seeing debug logs when using structlog. Because we weren’t passing along the log level. I suppose we could use airflow log level here?  Btw iirc, -v had no effect.

Reply:
So with this change, is there impact on both celery logging and airflow logging?

Reply:
this change just makes it so that the log level, when logging from structlog, is configured to whatevery is is set for celery

it doesn't change the log level for stdlib

you can test with this
```
from airflow.sdk.log import configure_logging
import structlog
import logging

configure_logging(output=sys.stdout.buffer)
l = structlog.get_logger()
l.debug("hi")
log = logging.getLogger()
log.debug("hi")
```

what you will find is, since configure_logging defaults to debug, after calling configure, structlog will be at debug, and stlib logging will be at info.  this to me seems undesirable doesn't it?

Reply:
also, if you would not set the log level from this config here, what would you set it to?  currently it defaults to debug